setup.py

```diff
@@ -1,11 +1,9 @@
-import sys
 import io
 from setuptools import setup
 
 
 requirements = ["requests>=2.20.0,<3.0"]
 
-# use io.open until python2.7 support is dropped
 with io.open("README.md", encoding="utf8") as f:
```

Comment:
Since the comment above was removed, drop `io` here and just use `open`? Looks like `open` is a built in function for Python 3 now.

Reply:
will do